<launch>
	<!-- ========================================================== -->
	<!-- Launch everything needed for strokelitude.                 -->
	<!-- ========================================================== -->


    <include file="$(find StrokelitudeROS)/launch/params_strokelitude.launch" ns="strokelitude" />
    <include file="$(find StrokelitudeROS)/launch/params_camera.launch"       ns="camera" />

    <!-- 
    <include file="$(find StrokelitudeROS)/launch/camera.launch" />
    -->
    <node name="playbag" pkg="rosbag" type="play" args="--clock --delay=1 --loop --rate=1 /home/ssafarik/strokelitude_fly2.bag" required="true" /> 
    <node name="view_mask"        pkg="image_view"     type="image_view"  ns="camera"  args="image:=/strokelitude/wba/image_mask" />
    
    <node name="strokelitude"           pkg="StrokelitudeROS"  type="cvgui_launchable.py"       />
    
</launch>
